lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -3293,6 +3293,11 @@ void __cdecl _alloca_probe_16()
 
         Assert(!(callInfo.Flags & CallFlags_New));
 
+        if (!JavascriptOperators::IsObject(args[0]))
```

Comment:
Indeed. I did feel unnatural to me as well. :) I didn't realize that args[0] is a Var and it could be directly passed to the IsCallable call.  

---
In reply to: [126813331](https://github.com/Microsoft/ChakraCore/pull/3320#discussion_r126813331) [](ancestors = 126813331,126812211)